vkconfig/configurator.h

```diff
@@ -174,6 +175,11 @@ class Configurator {
     int GetCustomLayersPathSize() const;
     const QString& GetCustomLayersPath(int path_index) const;
 
+    QStringList VK_LAYER_PATH;  // If this environment variable is set, this contains
```

Comment:
vk_layer_path in theory and private ideally so vk_layer_path_

Reply:
I think mirroring the environment variable name is more clear. Both in purpose and intent. We are allowed to depart from the typical naming convention if we have a good reason. I think this is a good reason.